workers/ohsome_quality_analyst/oqt.py

```diff
@@ -169,21 +186,45 @@ async def _create_indicator(
 
 
 @create_indicator.register
-async def _create_indicator(  # noqa
-    parameters: IndicatorBpolys,
-    force: bool = False,
-) -> Indicator:
+async def _create_indicator(parameters: IndicatorBpolys, *_args) -> Indicator:  # noqa
+    """Create an indicator from scratch."""
+    name = parameters.name.value
+    layer: Layer = from_dict(
+        data_class=LayerDefinition,
+        data=get_layer_definition(parameters.layer_name.value),
+    )
+    feature = parameters.bpolys
+
+    logging.info("Calculating Indicator for custom AOI ...")
+    logging.info("Indicator name: " + name)
+    logging.info("Layer name:     " + layer.name)
+
+    indicator_class = name_to_class(class_type="indicator", name=name)
+    indicator = indicator_class(layer, feature)
+
+    logging.info("Run preprocessing")
+    await indicator.preprocess()
+    logging.info("Run calculation")
+    indicator.calculate()
+    logging.info("Run figure creation")
+    indicator.create_figure()
+
+    return indicator
```

Comment:
This seems like almost a copy of below, maybe refactor this in the future. For now it's okay, I think.

Reply:
I stumbled across the duplication as well and thought about refactoring it out. But I did not want to introduce another function which "creates an Indicator". Then one needs to jump from on function to the next when reading the code for Indicator creation. 
If it were not for the logging messages this code would be quite concise as well. I thought about removing the logging statements or factor them out to some kind of decorator(?).

Reply:
related to https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=245&id=ohsome-quality-analyst&open=AX9QraTuoHkkEpvZxv0B

Reply:
Created an issue for addressing this later.
https://github.com/GIScience/ohsome-quality-analyst/issues/287